modules/op-resolver-tasks-step-actions-about.adoc

```diff
@@ -0,0 +1,65 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: CONCEPT
+[id="resolver-tasks-step-actions-about_{context}"]
+
+= About non-versioned and versioned tasks and step actions
+
+The `openshift-pipelines` namespace includes versioned tasks and step actions alongside standard non-versioned tasks and step actions. For example, installing the {pipelines-title} Operator v1.16 creates the following items:
+
+* `buildah-1-16-0` versioned task
+* `buildah` non-versioned task
+* `git-clone-1-16-0` versioned `StepAction` definition
+* `git-clone` non-versioned `StepAction` definition
+
+Non-versioned and versioned tasks and step actions have the same metadata, behavior, and specifications, including `params`, `workspaces`, and `steps`. However, they behave differently when you disable them or upgrade the Operator.
+
+Before adopting non-versioned or versioned tasks and step actions as a standard in production environments, cluster administrators might consider their advantages and disadvantages.
```

Comment:
```suggestion
Before adopting non-versioned or versioned tasks and step actions as a standard in production environments, cluster administrators should consider their advantages and disadvantages.
```
Hmm, is it _advisable_ to consider the differences? "Might" doesn't sound like advice, per se; seems very very optional. What's the happy path to direct customers to do?